### PR TITLE
feat: add labels and annotations for storage classes

### PR DESCRIPTION
**Pull request by fad3t**
Hello!

I would like to contribute a change that allows the customization of labels and annotations for the different storage classes. This also gives the option to set a custom name for the `VolumeSnapshotClass`.

Thanks,
Fred

### CODE REVIEW

**Comment by fad3t** [comment]
This fixes #80

File: charts/nutanix-csi-storage/Chart.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 name: nutanix-csi-storage
-version: 2.6.4
+version: 2.7.0
```
Comment (tuxtof) [inline charts/nutanix-csi-storage/Chart.yaml:3]: ok to bump version to 2.6.5

File: charts/nutanix-csi-storage/Chart.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 kubeVersion: ">= 1.17.0-0"
 description: Nutanix Container Storage Interface (CSI) Driver
 home: https://github.com/nutanix/helm
 maintainers:
   - name: nutanix-cloud-native-bot
     email: cloudnative@nutanix.com
 icon: https://avatars2.githubusercontent.com/u/6165865?s=200&v=4
-appVersion: "2.6.4"
+appVersion: "2.7.0"
```
Comment (tuxtof) [inline charts/nutanix-csi-storage/Chart.yaml:11]: driver will not change => 2.6.4

File: charts/nutanix-csi-storage/Chart.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ annotations:
   artifacthub.io/displayName: "Nutanix CSI Storage"
   artifacthub.io/containsSecurityUpdates: "true"
   artifacthub.io/changes: |
-    - Update Nutanix CSI driver to v2.6.4
+    - Update Nutanix CSI driver to v2.7.0
```
Comment (tuxtof) [inline charts/nutanix-csi-storage/Chart.yaml:26]: don't change this line

**Review by tuxtof — CHANGES_REQUESTED** [review]

**Comment by fad3t** [comment]
Thanks, version has been adapted.

**Pull request merged**